### ISSUE TITLE
geolocate_uavsar hanging

### ISSUE THREAD

**Issue by jacktarricone** [SnowEx/uavsar_pytools#34]
@ZachKeskinen @rpalomaki Issue with ```geolocate_uavsar```. Maybe test it out on your local machines and see if it will work? It produced the llh .tifs, but then isn't ingesting them to the ```geocodeUsingGdalWarp``` function.

```
%matplotlib inline
import sys
import os
import glob
import numpy as np
from osgeo import gdal
import matplotlib.pyplot as plt
import rasterio as rio
from rasterio.plot import show
from uavsar_pytools.georeference import geolocate_uavsar

## set function inputs

# input data
unw_data = '/Users/jacktarricone/jemez_isce/igrams/20200212T1545_20200226T1627/20200212T1545_20200226T1627.unw_snaphu.unw.vrt'

# annotationn file
ann_fp = '/Users/jacktarricone/jemez_isce/download/alamos_35915_20013_000_200226_L090HH_01_BU.ann'

# saving directory
out_dir = '/Users/jacktarricone/jemez_isce/igrams/20200212T1545_20200226T1627'

# lat lon heigh file
llh_fp = '/Users/jacktarricone/jemez_isce/download/alamos_35915_01_BU_s1_2x8.llh'

# run funciton
unw_geolocated = geolocate_uavsar(in_fp = unw_data, ann_fp = ann_fp, out_dir = out_dir, llh_fp = llh_fp)
print(unw_geolocated[0])
```

output 
<img width="833" alt="Screen Shot 2022-06-29 at 10 42 36 AM" src="https://user-images.githubusercontent.com/54866783/176501486-ad3b3a1c-5e5b-49f4-9ae3-1c621b1afbf9.png">

**Comment by rpalomaki** [comment]
@jacktarricone Can you upload the exact files you are using to a directory somewhere so we can test? Maybe a new folder at tests/data or something?

**Comment by jacktarricone** [comment]
@rpalomaki Data is on the google drive here - [https://drive.google.com/drive/u/1/folders/1tZyem2jSzmMuvnMu_v7m8WAJ15lu1weS]

**Comment by jacktarricone** [comment]
I broke ```geocodeUsingGdalWarp``` out into pieces and tried running each separately. Everything works until the ```gdal.warp``` at the end. Given we didn't change anything and it was working in uavsar_snow and my local nb, do we think this is possible a gdal bug? I'm going to try going down to past versions of gdal to see if it works.


<img width="772" alt="Screen Shot 2022-06-29 at 2 45 39 PM" src="https://user-images.githubusercontent.com/54866783/176550861-2c992752-045c-4d48-b5ee-647d27784e3e.png">

**Comment by jacktarricone** [comment]
Update - It finally ran but took 20 minutes. I'm not sure why given it used to take maybe 20 seconds with larger datasets.

Even when going back and rerunning the code in [http://localhost:8888/lab/tree/uavsar_snow/examples/carc_geolocate_test.ipynb](http://localhost:8888/lab/tree/uavsar_snow/examples/carc_geolocate_test.ipynb), it's still taking a long time. 

@ZachKeskinen did you change anything when importing into ```uavsar_pytools```? I wouldn't imagine so and therefore am caused what could be causing the slowdown..

**Comment by ZachKeskinen** [comment]
I don't think so, I think the only pieces I might have changed would be all file io so shouldn't affect geocoding.

versionwise it was working with gdal == 3.0.2 on my computer when we were testing.
